scalikejdbc-core/src/test/scala/BasicUsageSpec.scala

```diff
@@ -362,7 +362,7 @@ class BasicUsageSpec extends FlatSpec with Matchers with LoanPattern {
     try {
       TestUtils.initialize(tableName)
       DB localTx { implicit session =>
-        SQL("insert into " + tableName + " (id, name) values (999, 'Alice')").batchByName(Seq.empty: _*).apply()
+        SQL("insert into " + tableName + " (id, name) values (999, 'Alice')").batchByName(Seq.empty[Seq[(String, Any)]]: _*).apply()
```

Comment:
ambiguous overload :cry:

Should we rename or remove Symbol methods? 🤔

Reply:
In reality, we don't give such an ambiguous type values here. In my understanding, both of the following two examples work.

```scala
val paramss: Seq[(String, Any)] = ...
sql"....".batchByName(paramss: _*).apply()

val paramss: Seq[(Symbol, Any)] = ...
sql"....".batchByName(paramss: _*).apply()
```
At this moment, symbol literal is deprecated but `Symbol` type is not yet. My primary option would be supporting both string and symbol as your current code does.

Reply:
>  My primary option would be supporting both string and symbol as your current code does.

🙆‍♂